README.rst

```diff
@@ -1,32 +1,36 @@
+**WARNING**: This repository will eventually be obsolete. The content will
+exist under the `dwave-examples <https://github.com/dwave-examples>`_ account.
+
 Demos
 =====
 A collection of demos using the D-Wave Ocean SDK.
 
-**NOTE** Each demo is stored in its own subdirectory and may have its own
-set of package requirements (for example, `antenna-selection/requirements.txt
-<antenna-selection/requirements.txt>`_).
+**NOTE** Each demo is stored in its own submodule and may have its own
+set of package requirements (e.g. `antenna-selection/requirements.txt
+<https://github.com/dwave-examples/antenna-selection/blob/master/requirements.txt>`_).
 
 Setup Instructions
 ------------------
-As each demo may have a different set of package requirements, you may want to
-set up a `virtual environment <https://docs.ocean.dwavesys.com/en/latest/overview/install.html#python-virtual-environment>`_
+As each demo may have a different set of package requirements, you may
+want to set up a `virtual environment <https://docs.ocean.dwavesys.com/en/latest/overview/install.html#python-virtual-environment>`_
 to contain said packages and run your demo within that environment.
 
 Package Installation
 ~~~~~~~~~~~~~~~~~~~~
 ::
 
-  git clone https://github.com/dwavesystems/demos.git
-  cd demos/<desired-demo>
-  pip install -r requirements.txt
+  git clone --recursive https://github.com/dwavesystems/demos.git
+  cd demos/
+  git submodule foreach git pull origin master
```

Comment:
I don't think you need this (which is a rather new syntax anyway) if you clone with `--recursive` (also works on new git only).

Reply:
I believe I do need the `submodule foreach ..` because when I do a fresh clone with `--recursive`, I only get the submodules at the commit that this `demos` repo is associated to. In other words, if I don't use this command, I won't get the latest changes from the submodules.

This might be great for some repos, but for our use case, I believe we want to just grab the latest. (Alternatively, I could just constantly update this repo with the latest submodule commit ids but that would be insane, imho)

Reply:
Well that's the main point of submodules -- the point to a fixed commit!

If you want to inform users of this option (pull latest every), I would split that as a separate note. The problem otherwise is, when you do `git status` after this batch pull, you'll see submodule repos changed.

Reply:
Yup, made the separate note about how to update the submodules